e2e/cp_test.go

```diff
@@ -4196,3 +4197,33 @@ func TestCountingWriter(t *testing.T) {
 	expected := fs.Expected(t, fs.WithFile(filename, content, fs.WithMode(0644)))
 	assert.Assert(t, fs.Equal(cmd.Dir, expected))
 }
+
+// It should skip special files and don't try to upload them
+func TestUploadingSocketFile(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip()
+	}
+	t.Parallel()
+	s3client, s5cmd := setup(t)
+	bucket := s3BucketFromTestName(t)
+	createBucket(t, s3client, bucket)
+	workdir := fs.NewDir(t, t.Name())
+	defer workdir.Remove()
+	sockaddr := workdir.Path() + "/s5cmd.sock"
+	ln, err := net.Listen("unix", sockaddr)
+	if err != nil {
+		t.Log(err)
+	}
+	t.Cleanup(func() {
+		ln.Close()
+		os.Remove(sockaddr)
+	})
+	cmd := s5cmd("cp", sockaddr, "s3://"+bucket+"/")
+	result := icmd.RunCmd(cmd, withWorkingDir(workdir))
+	// assert no error
+	assertLines(t, result.Stderr(), map[int]compareFunc{})
```

Comment:
You can pass `nil` instead of an unallocated map;
```suggestion
	assertLines(t, result.Stderr(), nil)
```

Reply:
Same applies to other `assertLines` calls those not expecting any outputs.

Reply:
Done. https://github.com/peak/s5cmd/pull/618/commits/80b230986e595c30ee903d0153145b5400d9f60c